src/ServiceInsight/ValueConverters/SearchStringConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ServiceInsight.ValueConverters
+{
+    using System;
+    using System.Globalization;
+    using System.Windows.Data;
+
+    public class SearchStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var str = string.Format("{0}", value);
+            if (str.StartsWith("\"") && str.EndsWith("\""))
+            {
+                str = str.Substring(1, str.Length - 2);
+            }
+
+            return str;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return string.Format("\"{0}\"", value);
+        }
```

Comment:
I'm not a real fan of the exceptions. The converters are invoked by the databinding pipeline so any exceptions are [treated as application exceptions](https://msdn.microsoft.com/en-us/library/system.windows.data.ivalueconverter.convert.aspx?f=255&mspperror=-2147217396#Anchor_1) (could crash?)

What about:

```CSharp
public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
{
    if (targetType != typeof(string))
    {
        return DependencyProperty.UnsetValue;
    }

    var stringValue = value as string;
    if (stringValue == null)
    {
        return DependencyProperty.UnsetValue;
    }

    return stringValue.StartsWith("\"", true, culture) && stringValue.EndsWith("\"", true, culture)
        ? stringValue.Substring(1, stringValue.Length - 2)
        : stringValue;
}
```